src/test/java/uk/gov/hmcts/cmc/claimstore/services/ccd/callbacks/mediation/MediationFailedCallbackHandlerTest.java

```diff
@@ -71,14 +75,17 @@
         SampleClaim.getWithClaimantResponseRejectionForPartAdmissionAndMediation();
     private CallbackRequest callbackRequest;
 
+    //TODO Clean up these tests
```

Comment:
If this is expected to be done as part of a specific other story, please include the JIRA number in the comment